cross/dvb-apps/patches/build-libs-only.patch

```diff
@@ -0,0 +1,13 @@
+--- Makefile.orig	2017-03-27 21:10:20.023847566 +0200
++++ Makefile	2017-03-27 21:10:46.282317463 +0200
+@@ -10,8 +10,8 @@
+ 
+ all clean install:
+ 	$(MAKE) -C lib $@
+-	$(MAKE) -C test $@
+-	$(MAKE) -C util $@
++#	$(MAKE) -C test $@
```

Comment:
Patch will be simply shorter if lines removed

Reply:
Sorry, I don't understand. Do you mean that you want me to remove the lines instead of commenting them out?

Reply:
yes I do... less lines to maintain/review...